collectors/scm/github/src/main/java/com/capitalone/dashboard/collector/DefaultGitHubClient.java

```diff
@@ -721,4 +721,4 @@ private static String getTimeForApi (Date dt) {
     }
 }
 
-// X-RateLimit-Remaining
```

Comment:
Revert this file change

Reply:
@Sbrenthughes whats wrong with that file. i don't see any difference over there.